modules/network-observability-includelist-example.adoc

```diff
@@ -19,7 +19,7 @@ You can create custom Prometheus rules for the Netobserv dashboard metrics to tr
 [source,yaml]
 ----
 apiVersion: monitoring.coreos.com/v1
```

Comment:
The apiVersion has changed also:

```suggestion
apiVersion: monitoring.openshift.io/v1
```

However, while I'm able to create this alert, it doesn't seem to be picked up or triggered by the monitoring stack, if I refer to the Alert page in console ... I must be missing something...